index.js

```diff
@@ -1,19 +1,157 @@
 const toggleBtn = document.querySelector('#toggleBtn');
 const ulElem = document.querySelector('#menuList');
+const projectContainer = document.querySelector('#project-list');
+const modalContainer = document.querySelector('#detail');
+const modalMainContainer = document.querySelector('#detail1');
+const lists = document.querySelectorAll('#menuList li a');
+const closeBtn = document.querySelector('#btn_close');
+const blurGround = document.querySelector('.blurBg');
+const contactForm = document.getElementById('contact-form');
+const errorMsg = document.querySelector('#submitMessage');
 
 toggleBtn.addEventListener('click', () => {
   ulElem.style.display = 'block';
 });
 
-const closeBtn = document.querySelector('#btn_close');
 closeBtn.addEventListener('click', () => {
   ulElem.style.display = 'none';
 });
 
-const lists = document.querySelectorAll('#menuList li a');
-
 lists.forEach((list) => {
   list.addEventListener('click', () => {
     ulElem.style.display = 'none';
   });
 });
+
+const projectArray = [
+  {
+    name: 'Multi-Post Stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    img: '/images/placeholder.png',
+    technologies: ['css', 'html', 'javascript', 'Ruby'],
+    linkDemo: '',
+    sourceCode: 'https://github.com/til2to/porfolio_projects/tree/main',
+  },
```

Comment:
- 👏🏻 👏🏻 Excellent work implements an array of objects to store JavaScript objects containing the popup windows information for each project. 😉  💯